runtime/bindings/python/tests/test_ngraph/test_if.py

```diff
@@ -0,0 +1,171 @@
+
+
+def simple_if(condition_val):
+    condition = ov.constant(condition_val, dtype=np.bool)
+    # then_body
+    X_t = ov.parameter([2], np.float32, "X")
+    Y_t = ov.parameter([2], np.float32, "Y")
+
+    then_mul = ov.multiply(X_t, Y_t)
+    then_body_res_1 = ov.result(then_mul)
+    then_body = GraphBody([X_t, Y_t], [then_body_res_1])
+    then_body_inputs = [TensorIteratorInvariantInputDesc(1, 0), TensorIteratorInvariantInputDesc(2, 1)]
+    then_body_outputs = [TensorIteratorBodyOutputDesc(0, 0)]
+
+    # else_body
+    X_e = ov.parameter([2], np.float32, "X")
+    Y_e = ov.parameter([2], np.float32, "Y")
+    add_e = ov.add(X_e, Y_e)
+    else_body_res_1 = ov.result(add_e)
+    else_body = GraphBody([X_e, Y_e], [else_body_res_1])
+    else_body_inputs = [TensorIteratorInvariantInputDesc(1, 0), TensorIteratorInvariantInputDesc(2, 1)]
+    else_body_outputs = [TensorIteratorBodyOutputDesc(0, 0)]
+
+    X = ov.constant([3, 4], dtype=np.float32)
+    Y = ov.constant([2, 1], dtype=np.float32)
+    if_node = ov.if_op(condition, [X, Y], (then_body, else_body), (then_body_inputs, else_body_inputs),
+                       (then_body_outputs, else_body_outputs))
+    relu = ov.relu(if_node)
+    return relu
+
+
+def simple_if_without_parameters(condition_val):
+    condition = ov.constant(condition_val, dtype=np.bool)
+
+    # then_body
+    then_constant = ov.constant(0.7, dtype=np.float)
+    then_body_res_1 = ov.result(then_constant)
+    then_body = GraphBody([], [then_body_res_1])
+    then_body_inputs = []
+    then_body_outputs = [TensorIteratorBodyOutputDesc(0, 0)]
+
+    # else_body
+    else_const = ov.constant(9.0, dtype=np.float)
+    else_body_res_1 = ov.result(else_const)
+    else_body = GraphBody([], [else_body_res_1])
+    else_body_inputs = []
+    else_body_outputs = [TensorIteratorBodyOutputDesc(0, 0)]
+
+    if_node = ov.if_op(condition, [], (then_body, else_body), (then_body_inputs, else_body_inputs),
+                       (then_body_outputs, else_body_outputs))
+    relu = ov.relu(if_node)
+    return relu
+
+
+def check_results(results, expected_results):
+    assert len(results) == len(expected_results)
+    for id_result, res in enumerate(results):
+        assert np.allclose(res, expected_results[id_result])
+
+
+def check_if(if_model, cond_val, exp_results):
+    last_node = if_model(cond_val)
+    runtime = get_runtime()
+    computation = runtime.computation(last_node)
+    results = computation()
+    check_results(results, exp_results)
+
+
+def test_if():
+    check_if(simple_if, True, [np.array([6, 4], dtype=np.float32)])
+    check_if(simple_if, False, [np.array([5, 5], dtype=np.float32)])
+    check_if(simple_if_without_parameters, True, [np.array([0.7], dtype=np.float32)])
+    check_if(simple_if_without_parameters, False, [np.array([9.0], dtype=np.float32)])
+
+    # After deleting evalute method for if, constant folding stopped working.
+    # As result bug with id 67255 began to appear
+    # xfail(67255)
```

Comment:
why these tests were commented? can we just mark them as xfail?
Something like that: pytest.mark.xfail(...)

Reply:
done